week1/scary/stites/Main.hs

```diff
@@ -0,0 +1,82 @@
+---------------------------------------------------------------------------------
+-- Scary words
+--
+-- If you assign numbers to letters (A=1, B=2, ..., Z=26), then a word is scary
+-- if the sum of its letters is 13. “baaed”, for instance, is scary (especially
+-- when at first you don't understand it's a silly verb and think it's an ancient
+-- god's name).
+--
+-- Find all scary words in the `words` file (it's usually in
+-- `/usr/share/dict/words` or `/usr/dict/words`).
+--
+-- Common mistakes:
+--
+-- * Treating `zip's` as scary (the mistake is in assigning a numbers to *all*
+-- characters, not just non-letter ones, and then `'` usually gets a negative
+-- code).
+--
+-- * Treating `Iraq` as scary (uppercase characters should be treated the same as
+-- lowercase ones).
+---------------------------------------------------------------------------------
+{-# LANGUAGE OverloadedLists #-}
+module Main where
+
+import Data.Char
+import Data.List
+import qualified Data.HashSet as H
+import qualified Data.Vector as V
+
+main :: IO ()
+main = do
+  file <- readFile "../../../data/words"
+  putStrLn "Scary words:"
+  putStrLn . intercalate "," . filter isScary . lines $ file
+  return ()
```

Comment:
You don't need `return ()` here as `putStrLn` already has the type `IO ()` and thus returns `()`.